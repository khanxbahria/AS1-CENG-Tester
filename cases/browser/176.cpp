#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Browser* brows = new Browser();
Window a;
Window b;
Window c;
Window d;
Tab A = Tab("d", "jjj","ff");
Tab B = Tab("ddd", "kk","y");
Tab C = Tab("sss", "tt","i");
Tab D = Tab("jjj", "ccc","i");
Tab E = Tab("kk", "xxx","jjj");
Tab F = Tab("p", "y","lll");
Tab G = Tab("y", "g","e");
Tab H = Tab("qqq", "e","ff");
Tab I = Tab("kk", "c","vv");
Tab J = Tab("f", "q","k");
Tab K = Tab("rr", "nnn","l");
Tab L = Tab("a", "x","vv");
Tab M = Tab("lll", "k","eee");
Tab N = Tab("sss", "dd","cc");
Tab O = Tab("nn", "oo","rr");
Tab P = Tab("ww", "zz","rr");
Tab Q = Tab("kkk", "iii","a");
Tab R = Tab("ddd", "aa","cc");
Tab S = Tab("nnn", "k","rrr");
Tab T = Tab("nn", "sss","rr");
Tab U = Tab("zz", "rrr","z");
Tab V = Tab("m", "bb","aa");
Tab W = Tab("iii", "yyy","fff");
Tab X = Tab("w", "pp","sss");
Tab Y = Tab("kkk", "q","u");
Tab Z = Tab("ee", "t","gg");
a.changeActiveTabTo(1);
b.moveActiveTabTo(1);
c.changeActiveTabTo(1);
d.changeActiveTabTo(1);
a.newTab(A);
b.moveActiveTabTo(3);
c.moveActiveTabTo(3);
d.newTab(B);
a.newTab(C);
b.changeActiveTabTo(1);
c.moveActiveTabTo(1);
d.moveActiveTabTo(1);
a.newTab(D);
b.moveActiveTabTo(0);
c.changeActiveTabTo(0);
d.changeActiveTabTo(0);
a.newTab(E);
b.changeActiveTabTo(2);
c.moveActiveTabTo(2);
d.changeActiveTabTo(2);
a.changeActiveTabTo(1);
b.moveActiveTabTo(1);
c.newTab(F);
d.moveActiveTabTo(1);
a.moveActiveTabTo(3);
b.moveActiveTabTo(3);
c.newTab(G);
d.changeActiveTabTo(3);
a.newTab(H);
b.moveActiveTabTo(5);
c.moveActiveTabTo(5);
d.newTab(I);
a.moveActiveTabTo(1);
b.newTab(J);
c.moveActiveTabTo(1);
d.changeActiveTabTo(1);
a.moveActiveTabTo(1);
b.newTab(K);
c.changeActiveTabTo(1);
d.changeActiveTabTo(1);
a.changeActiveTabTo(5);
b.newTab(L);
c.moveActiveTabTo(5);
d.moveActiveTabTo(5);
a.moveActiveTabTo(5);
b.changeActiveTabTo(5);
c.moveActiveTabTo(5);
d.changeActiveTabTo(5);
a.moveActiveTabTo(3);
b.newTab(M);
c.newTab(N);
d.newTab(O);
a.changeActiveTabTo(3);
b.changeActiveTabTo(3);
c.newTab(P);
d.newTab(Q);
a.moveActiveTabTo(4);
b.moveActiveTabTo(4);
c.newTab(R);
d.newTab(S);
a.moveActiveTabTo(3);
b.changeActiveTabTo(3);
c.newTab(T);
d.moveActiveTabTo(3);
a.moveActiveTabTo(2);
b.changeActiveTabTo(2);
c.newTab(U);
d.moveActiveTabTo(2);
a.newTab(V);
b.changeActiveTabTo(1);
c.moveActiveTabTo(1);
d.moveActiveTabTo(1);
a.newTab(W);
b.changeActiveTabTo(5);
c.newTab(X);
d.moveActiveTabTo(5);
a.changeActiveTabTo(1);
b.changeActiveTabTo(1);
c.moveActiveTabTo(1);
d.moveActiveTabTo(1);
a.changeActiveTabTo(0);
b.changeActiveTabTo(0);
c.moveActiveTabTo(0);
d.newTab(Y);
a.newTab(Z);
b.newTab(Z);
c.newTab(Z);
d.moveActiveTabTo(2);
b.changeActiveTabTo(2);
c.moveActiveTabTo(2);
a.moveActiveTabTo(1);
b.changeActiveTabTo(1);
d.changeActiveTabTo(1);
a.moveActiveTabTo(3);
c.changeActiveTabTo(3);
d.moveActiveTabTo(3);
a.changeActiveTabTo(0);
b.moveActiveTabTo(0);
c.changeActiveTabTo(0);
d.moveActiveTabTo(0);
a.changeActiveTabTo(3);
c.changeActiveTabTo(3);
d.moveActiveTabTo(3);
b.changeActiveTabTo(3);
d.moveActiveTabTo(3);
a.moveActiveTabTo(1);
b.moveActiveTabTo(1);
d.changeActiveTabTo(1);
a.changeActiveTabTo(3);
a.changeActiveTabTo(4);
b.moveActiveTabTo(4);
c.moveActiveTabTo(4);
d.changeActiveTabTo(4);
a.changeActiveTabTo(1);
b.moveActiveTabTo(1);
c.changeActiveTabTo(1);
d.changeActiveTabTo(1);
a.moveActiveTabTo(1);
d.moveActiveTabTo(1);
a.changeActiveTabTo(4);
b.changeActiveTabTo(4);
c.changeActiveTabTo(4);
d.moveActiveTabTo(4);
a.changeActiveTabTo(1);
b.moveActiveTabTo(1);
c.changeActiveTabTo(1);
d.changeActiveTabTo(4);
b.moveActiveTabTo(3);
c.changeActiveTabTo(3);
d.moveActiveTabTo(3);
a.changeActiveTabTo(2);
b.moveActiveTabTo(2);
c.changeActiveTabTo(2);
b.moveActiveTabTo(5);
c.moveActiveTabTo(5);
a.moveActiveTabTo(5);
b.changeActiveTabTo(5);
d.moveActiveTabTo(5);
a.moveActiveTabTo(1);
a.moveActiveTabTo(2);
a.moveActiveTabTo(5);
b.changeActiveTabTo(5);
c.moveActiveTabTo(5);
a.changeActiveTabTo(4);
b.changeActiveTabTo(4);
c.changeActiveTabTo(4);
d.moveActiveTabTo(4);
a.moveActiveTabTo(4);
c.changeActiveTabTo(4);
a.changeActiveTabTo(3);
b.moveActiveTabTo(3);
c.moveActiveTabTo(3);
a.changeActiveTabTo(3);
b.changeActiveTabTo(3);
c.moveActiveTabTo(3);
d.changeActiveTabTo(3);
a.changeActiveTabTo(2);
c.moveActiveTabTo(2);
d.changeActiveTabTo(2);
a.changeActiveTabTo(4);
c.changeActiveTabTo(4);
brows->addWindow(a);
brows->addWindow(b);
brows->addWindow(c);
brows->addWindow(d);
brows->getWindow(2);
brows->getWindow(1);
brows->newWindow();
brows->getWindow(2);
brows->getWindow(3);
brows->mergeAllWindows();
brows->switchToWindow(9);
brows->print();
brows->mergeAllWindows();
brows->newWindow();
brows->closeWindow();
brows->closeWindow();
brows->getWindow(1);
brows->closeWindow();
brows->closeEmptyWindows();
brows->getWindow(5);
brows->closeWindow();
brows->closeEmptyWindows();
brows->closeWindow();
brows->mergeAllWindows();
brows->newWindow();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->print();
brows->closeWindow();
brows->newWindow();
brows->getWindow(8);
brows->getWindow(3);
brows->closeEmptyWindows();
brows->newWindow();
brows->switchToWindow(5);
brows->newWindow();
brows->newWindow();
brows->print();
brows->closeWindow();
brows->getWindow(2);
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->switchToWindow(4);
brows->mergeAllWindows();
brows->print();
brows->closeWindow();
brows->getWindow(8);
brows->closeEmptyWindows();
brows->switchToWindow(3);
brows->newWindow();
brows->getWindow(1);
brows->print();
brows->switchToWindow(6);
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->newWindow();
brows->closeEmptyWindows();
brows->getWindow(3);
brows->print();
brows->getWindow(2);
brows->getWindow(3);
brows->print();
brows->print();
brows->print();
brows->mergeAllWindows();
brows->switchToWindow(0);
brows->getWindow(0);
brows->closeEmptyWindows();
brows->switchToWindow(3);
brows->getWindow(8);
brows->getWindow(9);
brows->getWindow(1);
brows->print();
brows->closeEmptyWindows();
brows->closeWindow();
brows->switchToWindow(1);
brows->newWindow();
brows->switchToWindow(1);
brows->getWindow(8);
brows->switchToWindow(8);
brows->mergeAllWindows();
brows->getWindow(7);
brows->closeEmptyWindows();
brows->newWindow();
brows->closeWindow();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->print();
brows->getWindow(7);
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->print();
brows->closeWindow();
brows->newWindow();
brows->print();
brows->getWindow(4);
brows->getWindow(3);
brows->closeWindow();
brows->newWindow();
brows->newWindow();
brows->closeEmptyWindows();
brows->switchToWindow(3);
brows->closeEmptyWindows();
brows->getWindow(7);
brows->closeEmptyWindows();
brows->switchToWindow(7);
brows->switchToWindow(3);
brows->switchToWindow(0);
brows->switchToWindow(1);
brows->getWindow(7);
brows->newWindow();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->closeWindow();
delete brows;
}
	return 0;
}