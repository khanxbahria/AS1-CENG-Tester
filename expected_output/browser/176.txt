The active tab is tab 33












t
ee
gg

xxx
kk
jjj

bb
m
aa

yyy
iii
fff

e
qqq
ff

tt
sss
i

ccc
jjj
i

jjj
d
ff

t
ee
gg

nnn
rr
l





x
a
vv

q
f
k

k
lll
eee

pp
w
sss





g
y
e

y
p
lll

rrr
zz
z

t
ee
gg

dd
sss
cc

zz
ww
rr

aa
ddd
cc

sss
nn
rr

kk
ddd
y

c
kk
vv

k
nnn
rrr





q
kkk
u

oo
nn
rr

iii
kkk
a

Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 2












Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 1








The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




